kornia/feature/laf.py

```diff
@@ -115,19 +115,40 @@ def set_laf_orientation(LAF: torch.Tensor, angles_degrees: torch.Tensor) -> torc
     return laf_out
 
 
-def laf_from_center_scale_ori(xy: torch.Tensor, scale: torch.Tensor, ori: torch.Tensor) -> torch.Tensor:
+def laf_from_center_scale_ori(xy: torch.Tensor,
+                              scale: Optional[torch.Tensor] = None,
+                              ori: Optional[torch.Tensor] = None) -> torch.Tensor:
     """Return orientation of the LAFs, in radians. Useful to create kornia LAFs from OpenCV keypoints.
 
     Args:
         xy: tensor [BxNx2].
-        scale: tensor [BxNx1x1].
-        ori: tensor [BxNx1].
+        scale: tensor [BxNx1x1]. If not provided, scale = 1 is assumed
+        ori: tensor [BxNx1]. If not provided orientation = 0 is assumed
 
     Returns:
         tensor BxNx2x3.
     """
     names = ['xy', 'scale', 'ori']
-    for var_name, var, req_shape in zip(names, [xy, scale, ori], [("B", "N", 2), ("B", "N", 1, 1), ("B", "N", 1)]):
+    xy_req_shape = ("B", "N", 1, 1)
+    if not isinstance(xy, torch.Tensor):
```

Comment:
I recently added this simple api to simplify checkings.
https://github.com/kornia/kornia/blob/master/kornia/enhance/adjust.py#L340-L341

not a requirement but looks more clean
also if time, isolate torch imports

Reply:
Cool, I will do refactor with this checks later. They do look well, and also I might improve docs at that time.

Reply:
Great